test/e2e/api_budget_test.go

```diff
@@ -0,0 +1,30 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	e2eutil "github.com/openshift/hypershift/test/e2e/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestAPIBudget(t *testing.T) {
```

Comment:
What's the purpose of this one?

/hold

Reply:
gah, how did that get included! :confounded: